assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/applicationContext-spring-security-csrf.xml

```diff
@@ -64,6 +64,25 @@
         -->
         <swm:regex-request-matcher pattern="^/api/licenseManager/addLicense\b.*" methods="POST" />
 
+        <!--
+          POST /api/scheduler/job
+        -->
+        <swm:regex-request-matcher pattern="^/api/scheduler/job\b.*" methods="POST" />
+
+        <!--
+          POST /api/scheduler/job/update
+        -->
+        <swm:regex-request-matcher pattern="^/api/scheduler/job/update\b.*" methods="POST" />
```

Comment:
I believe the regular expression of the first request matcher "shadows" that of the second.
You could use the first only, and, in the XML comment, put the two URLs that would match.
I checked and there appear to be no other endpoints in the `/api/scheduler` space that would match.

Reply:
changed as requested